### PR TITLE
APIGOV-28759 - update readme

### PR DESCRIPTION
**Pull request by jcollins-axway**

### CODE REVIEW

**Comment by jcollins-axway** [comment]
@robertengaxway could you look this over, please

**Review by sbolosan — APPROVED** [review]

File: README_traceability.md [hunk 1]
```diff
@@ -16,21 +16,21 @@ docker run --env-file env_vars -v `pwd`/keys:/keys ghcr.io/axway/mulesoft_tracea
...
+| MULESOFT_SSL_MAXVERSION         | mulesoft.ssl.maxVersion         | String value for the maximum SSL/TLS version that is acceptable. If empty, then the maximum version supported by this package is used, which is currently TLS 1.3. Allowed values are: TLS1.0, TLS1.1, TLS1.2, TLS1.3                                                                        | Internally, this value defaults to empty                                                                                                                                          |
+| MULESOFT_SSL_MINVERSION         | mulesoft.ssl.minVersion         | String value for the minimum SSL/TLS version that is acceptable. If zero, empty TLS 1.0 is taken as the minimum. Allowed values are: TLS1.0, TLS1.1, TLS1.2, TLS1.3                                                                                                                          | Internally, the value defaults toTLS1.2                                                                                                                                           |
```
Comment (sbolosan) [inline README_traceability.md:35]: unless its not rendering right for me in this view, there needs to be a space between to and TLS1.2 (toTLS1.2)

File: README_discovery.md [hunk 2]
```diff
@@ -16,24 +16,24 @@ docker run --env-file env_vars -v `pwd`/keys:/keys ghcr.io/axway/mulesoft_discov
...
+| MULESOFT_SSL_MAXVERSION         | mulesoft.ssl.maxVersion         | String value for the maximum SSL/TLS version that is acceptable. If empty, then the maximum version supported by this package is used, which is currently TLS 1.3. Allowed values are: TLS1.0, TLS1.1, TLS1.2, TLS1.3                                                                        | Internally, this value defaults to empty                                                                                                                                          |
+| MULESOFT_SSL_MINVERSION         | mulesoft.ssl.minVersion         | String value for the minimum SSL/TLS version that is acceptable. If zero, empty TLS 1.0 is taken as the minimum. Allowed values are: TLS1.0, TLS1.1, TLS1.2, TLS1.3                                                                                                                          | Internally, the value defaults toTLS1.2                                                                                                                                           |
```
Comment (sbolosan) [inline README_discovery.md:38]: here toTLS1.2 -> to TLS1.2

**Review by robertengaxway — APPROVED** [review]
Approved assuming Shane's edits are done

**Review by sbolosan — APPROVED** [review]

**Pull request merged**